Segment-Amplitude.podspec

```diff
@@ -21,6 +21,6 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Pod/Classes/**/*'
 
-  s.dependency 'Analytics', '~> 3.6' # support for iOS 7 +
-  s.dependency 'Amplitude-iOS', '~> 4.0'
+  s.dependency 'Analytics'
+  s.dependency 'Amplitude-iOS'
 end
```

Comment:
@OllaAshour can we actually add the specific upgraded target versions back here.  The` Example/Podfile.lock` is just keeping track of what's there, but if a customer does a pod install, that lock isn't going to be looked at and will be overwritten by the latest version which will be pulled by the `.podspec`  unless we indicate a target version for the dependency. The path of just taking the "latest" (not specifying version) is leaving us open to incompatibilities when a partner makes a backwards incompatible change to their SDK.